src/lib/isTaxID.js

```diff
@@ -852,6 +852,93 @@ function plPlCheck(tin) {
   return checksum === parseInt(tin[10], 10);
 }
 
+/*
+* pt-BR validation function
+* (Cadastro de Pessoas Físicas (CPF, persons)
+* Cadastro Nacional de Pessoas Jurídicas (CNPJ, entities)
+* Both inputs will be validated
+*/
+
+function ptBrCheck(tin) {
+  tin = tin.replace(/[^\d]+/g, '');
+  if (tin === '') return false;
+
+  if (tin.length === 11) {
+    let sum;
+    let ramainder;
+    sum = 0;
+    tin = tin.replace(/[^\d]+/g, '');
+
+    if ( // Reject known invalid CPFs
+      tin === '11111111111' ||
+      tin === '22222222222' ||
+      tin === '33333333333' ||
+      tin === '44444444444' ||
+      tin === '55555555555' ||
+      tin === '66666666666' ||
+      tin === '77777777777' ||
+      tin === '88888888888' ||
+      tin === '99999999999' ||
+      tin === '00000000000'
+    ) return false;
```

Comment:
This line is not covered by tests, you probably just need to add a testcase with an invalid CPF